flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/TableEnvHiveConnectorTest.java

```diff
@@ -124,9 +124,15 @@ public void testGetNonExistingFunction() throws Exception {
 	public void testDifferentFormats() throws Exception {
 		String[] formats = new String[]{"orc", "parquet", "sequencefile", "csv"};
 		for (String format : formats) {
-			if (format.equals("orc") && HiveShimLoader.getHiveVersion().startsWith("2.0")) {
-				// Ignore orc test for Hive version 2.0.x for now due to FLINK-13998
-				continue;
+			if (format.equals("orc")) {
+				String version = HiveShimLoader.getHiveVersion();
+				// hive 2.0.x with orc can not run in local mode, because Hadoop23Shims only
```

Comment:
Actually we could make 2.0.x read ORC tables in local FS, but writing to ORC tables didn't work. More details in https://issues.apache.org/jira/browse/FLINK-13998

Reply:
Got it, restore this code.